src/common/commonutils/UserUtils.c

```diff
@@ -185,6 +184,16 @@ static bool IsUserNonLogin(SimplifiedUser* user)
     return noLogin;
 }
 
+static bool IsSystemAccount(SimplifiedUser* user)
+{
+    return (user && ((user->username && (0 == strcmp(user->username, g_root))) || IsUserNonLogin(user) || (user->userId < 1000))) ? true : false;
```

Comment:
wouldn't it be simpler to just check if `userId < 1000` wouldn't that denote a system account? Why also check username + IsUserNonLogin?

Reply:
It's more solid this way. There may be special cases that do not respect just one rule, we need to be prepared for all that we can,